Products/CMFPlone/browser/templates/plone-frontpage.pt

```diff
@@ -39,7 +39,7 @@ Before you start exploring your newly created Plone site, please do the followin
 <p>After that, we suggest you do one or more of the following:</p>
 
 <ul>
-    <li>Find out <a class="link-plain" href="https://plone.com/features/" target="_blank">What's new in Plone</a>.</li>
+    <li>Find out <a class="link-plain" href="https://plone.org/what-is-plone/plone" target="_blank">What's new in Plone</a>.</li>
```

Comment:
Let's change this completely. That link is not useful because it is marketing to someone who has already installed Plone and does not need to be sold on it again, and the page name no longer exists.

```suggestion
    <li>Get the latest <a class="link-plain" href="https://plone.org/what-is-plone/plone" target="_blank">news</a> about Plone.</li>
```

Reply:
I'll do this one... This was my initial thought too but later decided not to change!